fili-core/src/test/java/com/yahoo/bard/webservice/data/config/names/TestDimensionName.java

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.config.names;
+
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+/**
+ * Bard test dimensions.
+ */
+public enum TestDimensionName implements DimensionName {
+    NAME,
+    AGE,
+    ETHNICITY,
+    GENDER;
```

Comment:
Move the semicolon to the next line so that when we add things to the enum list in the future, the last item won't be shown as deleted and then re-added due to moving semicolon to the next item. Apply to other files also.